vyos/vyos-build
# Kernel: T4599: set CONFIG_VIRTIO_FS to m

## Change Summary

Enable virtiofs to support lxd-agent.
See https://github.com/canonical/lxd/blob/005bd8d473002a1f72b19b3e4f9e05557bece639/lxd/instance/drivers/driver_qemu.go#L2510-L2549

## Types of changes


## Related Task(s)

* https://vyos.dev/T4599

## Component(s) name

- kernel

## Proposed changes

I would like to set CONFIG_VIRTIO_FS to m, because virtiofs or 9p is required by lxd-agent.

## How to test

```
cd ./packages/linux-kernel/
git clone --depth 1 -b v6.1.42 https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
./build-kernel.sh
cd ../..
sudo ./build-vyos-image --architecture amd64 --build-by "YOUR_EMAIL_ADDRESS" --build-type development iso
```

Boot from the ISO, then you can confirm the virtiofs module is enabled.

## Checklist:
